### PR TITLE
Two pointer implementation

### PR DESCRIPTION
**Pull request by jjliewie**
_If any of the below doesn't apply to this Pull Request, mark the checkbox as done._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions), which includes the following: 
  - I understand that if it is clear that I have not attempted to follow these guidelines (ex. if I have not used tabs to indent), my PR will be closed.
  - If changes are requested, I will re-request the review after making them.

### CODE REVIEW

File: content/3_Silver/More_Prefix_Sums.mdx [hunk 1]
```diff
@@ -218,6 +218,29 @@ public class MaxSubarraySumKadane {
 ```
 </JavaSection>
 
+<PySection>
+
+```python
+n = int(input())
+x = [0]*n
```
Comment (nwatx) [inline content/3_Silver/More_Prefix_Sums.mdx:225]:
```suggestion
x = [0] * n
```

File: content/3_Silver/Two_Pointers.mdx [hunk 2]
```diff
@@ -275,6 +275,39 @@ public class Books {
 
 </JavaSection>
 
+<PySection>
+
+```python
+n = int(input())
+t = int(input())
+
+books = [0]*n
```
Comment (nwatx) [inline content/3_Silver/Two_Pointers.mdx:284]:
```suggestion
books = [0] * n
```

**Review by nwatx — APPROVED** [review]
Both of these look good (although next time try to make two prs)

**Review by SansPapyrus683 — CHANGES_REQUESTED** [review]
tabs

**Comment by jjliewie** [comment]
> tabs

done

**Comment by SansPapyrus683** [comment]
why close?

**Comment by jjliewie** [comment]
> why close?

i fixed the format

**Comment by SansPapyrus683** [comment]
I think you closed it loo

**Comment by SansPapyrus683** [comment]
Wait nvm wrong comment

File: content/3_Silver/More_Prefix_Sums.mdx [hunk 3]
```diff
@@ -218,6 +218,29 @@ public class MaxSubarraySumKadane {
...
+
+```python
+n = int(input())
+x = [0] * n
+for i in range(n):
+	x[i] = int(input())
+current_sum = x[0]
+max_subarray_sum = x[0]
+for i in range(1, n):
+	'''
```
Comment (SansPapyrus683) [inline content/3_Silver/More_Prefix_Sums.mdx:231]: use double quotes for multiline comments

File: content/3_Silver/More_Prefix_Sums.mdx [hunk 4]
```diff
@@ -218,6 +218,29 @@ public class MaxSubarraySumKadane {
...
 
+<PySection>
+
+```python
+n = int(input())
+x = [0] * n
+for i in range(n):
+	x[i] = int(input())
+current_sum = x[0]
+max_subarray_sum = x[0]
```
Comment (SansPapyrus683) [inline content/3_Silver/More_Prefix_Sums.mdx:229]: use whitespace to separate relevant blocks of code like in the java/cpp sols

File: content/3_Silver/Two_Pointers.mdx [hunk 5]
```diff
@@ -275,6 +275,39 @@ public class Books {
...
+```python
+n = int(input())
+t = int(input())
+
+books = [0] * n
+
+for i in range(n):
+	books[i] = int(input())
+
+right, left, cur, ans = 0, 0, 0, 0
```
Comment (SansPapyrus683) [inline content/3_Silver/Two_Pointers.mdx:289]: put declarations on sep lines plz

**Review by SansPapyrus683 — CHANGES_REQUESTED** [review]
overall very nice

File: content/3_Silver/Two_Pointers.mdx [hunk 6]
```diff
@@ -274,7 +274,41 @@ public class Books {
...
+<PySection>
+
+```py
+n = int(input())
+t = int(input())
+
+books = [0] * n
+
+for i in range(n):
+	books[i] = int(input())
```
Comment (ryanchou-dev) [inline content/3_Silver/Two_Pointers.mdx:286]: probably better as:

```
books = list(map(int, input().split()))
```

**Review by ryanchou-dev — APPROVED** [review]
looks ok. i did most of sans' suggestions, since they're all minor

if you disagree with anything, lmk. also, happy first contribution 🎉

File: content/3_Silver/Two_Pointers.mdx (hunk 6 above)
Comment (jjliewie) [reply content/3_Silver/Two_Pointers.mdx]: Aren't the inputs in multiple lines? I changed it to books = [int(input()) for _ in range(n)]. sorry if I'm wrong.

**Review by SansPapyrus683 — APPROVED** [review]

File: content/3_Silver/Two_Pointers.mdx (hunk 6 above)
Comment (ryanchou-dev) [reply content/3_Silver/Two_Pointers.mdx]: oops sorry, you're right. thanks!

File: content/3_Silver/Two_Pointers.mdx [hunk 7]
```diff
@@ -274,7 +274,38 @@ public class Books {
 ```
 
 </JavaSection>
-
+<PySection>
+
+```py
+n = int(input())
+t = int(input())
```
Comment (bqi343) [inline content/3_Silver/Two_Pointers.mdx:281]: note: this code doesn't actually work on the sample ...

**Pull request merged**